### PR TITLE
Gfal2.19 & Davix 0.7.6

### PR DESCRIPTION
**Pull request by chaen**
I know these are not released version (although gfal2 will be released as is next week; davix should go in couple of weeks) but they fix segfault and https features we need (e.g https://its.cern.ch/jira/projects/DMC/issues/DMC-1227)

Tested in https://gitlab.cern.ch/CLICdp/iLCDirac/diracos-test/-/pipelines/2268292

@andresailer @petricm @chrisburr care to review/approve/merge ? 

BEGINRELEASENOTES

CHANGE: gfal2.17.1 -> gfal2.19.0 (final RC)
CHANGE: davix 0.7.5 -> davix 0.7.6 devel 
NEW: possibility to run mock rebuild without cleaning the environment

ENDRELEASENOTES

### CODE REVIEW

File: config/diracos.json [hunk 1]
```diff
@@ -472,14 +472,17 @@
             "name": "davix",
             "packages": [
                {
-                  "src": "https://diracos.web.cern.ch/diracos/SRPM/davix-0.7.5-1.el6.src.rpm",
-                  "originalRepo": "https://dl.fedoraproject.org/pub/epel/6/SRPMS/Packages/",
+                  "src": "https://diracos.web.cern.ch/diracos/SRPM/davix-0.7.6.200.3c48eab-1.el6.src.rpm ",
```
Comment (chrisburr) [inline config/diracos.json:475]:
```suggestion
                  "src": "https://diracos.web.cern.ch/diracos/SRPM/davix-0.7.6.200.3c48eab-1.el6.src.rpm",
```

File: routines/davix.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+from diracos import diracoslib
+import os
+import glob
+import tarfile
+import tempfile
+from yum import rpmUtils
+
+# TODO: factorize with glite-ce-cream-cli.py, since it is essentialy the same thing
+
+CMAKE3_SLC6_INSTALLER = 'https://github.com/Kitware/CMake/releases/download/v3.19.3/cmake-3.19.3-Linux-x86_64.sh'
```
Comment (petricm) [inline routines/davix.py:13]:
```suggestion
CMAKE3_SLC6_INSTALLER = 'https://diracos.web.cern.ch/diracos/bootstrap/cmake-3.19.3-Linux-x86_64.sh'
```
Just to be safe have in "in house"

File: config/diracos.json (hunk 1 above)
Comment (chaen) [reply config/diracos.json:475]: Thanks !

File: routines/davix.py (hunk 2 above)
Comment (chaen) [reply routines/davix.py:13]: good point !

**Review by chrisburr — APPROVED** [review]

**Pull request merged**